lighthouse-core/closure/closure-type-checking.js

```diff
@@ -32,14 +32,20 @@ gulp.task('compile-report', () => {
   return gulp.src([
     // externs
     'closure/third_party/commonjs.js',
-    'lib/filer-namer.js',
+
+    'lib/file-namer.js',
     'report/v2/renderer/*.js',
   ])
 
   // Ignore `module.exports` and `self.ClassName = ClassName` statements.
   .pipe(replace(/^\s\smodule\.exports = \w+;$/gm, ';'))
   .pipe(replace(/^\s\sself\.(\w+) = \1;$/gm, ';'))
 
+  // Remove node-specific code from file-namer so it can be included in report.
```

Comment:
kind of dumb to do such a custom solution for just this file, but since this is the only file that's completely intended to live in both node and the browser (all the other renderer files are only used in node for testing), I went with it. If/when we have more files that need to live in both worlds we can add a more general solution.